ignite/contrib/metrics/cohen_kappa.py

```diff
@@ -0,0 +1,84 @@
+from typing import Callable
+
+import torch
+
+from ignite.metrics import EpochMetric
+
+
+def non_weighted_cohen_kappa_compute_fn(y_preds: torch.Tensor, y_targets: torch.Tensor) -> float:
+    try:
```

Comment:
Please, put this check inside `CohenKappa.__init__` and let's create a single function getter using the wrapper as I proposed previously. Thanks

Reply:
Okay i will close this pull request and create a new one with the updated implementation. Thanks

Reply:
You can commit here as well. Let's avoid closing and opening new PRs for updates.